src/Mod/Draft/DraftGui.py

```diff
@@ -131,7 +131,7 @@ def __init__(self,widget,extra=None):
         else:
             self.form = widget
     def getStandardButtons(self):
-        return int(QtWidgets.QDialogButtonBox.Close)
+        return QtWidgets.QDialogButtonBox.Close.value
```

Comment:
In the documentation for both [Qt5](https://doc.qt.io/qtforpython-5/PySide2/QtWidgets/QWidget.html#PySide2.QtWidgets.PySide2.QtWidgets.QWidget.close) and [Qt6](https://doc.qt.io/qtforpython-6/PySide6/QtWidgets/QWidget.html#PySide6.QtWidgets.QWidget.close) I find that the return value of `.Close` is bool, but bool doesn't have a `.value` attribute. Am I missing something here?

Reply:
I can't explain it yet, but this is what I get
```
$ ipython                                                         
Python 3.12.2 (main, Feb 21 2024, 00:00:00) [GCC 13.2.1 20231205 (Red Hat 13.2.1-6)]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.22.2 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import PySide6.QtWidgets as QtWidgets

In [2]: int(QtWidgets.QDialogButtonBox.Close)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[2], line 1
----> 1 int(QtWidgets.QDialogButtonBox.Close)

TypeError: int() argument must be a string, a bytes-like object or a real number, not 'StandardButton'

In [3]: QtWidgets.QDialogButtonBox.Close.value
Out[3]: 2097152

In [6]: import PySide6
In [7]: PySide6.__version__
Out[7]: '6.6.2'
```

Reply:
I was looking at the wrong places in the documentation, here are the correct ones [Qt5 ](https://doc.qt.io/qtforpython-5/PySide2/QtWidgets/QDialogButtonBox.html#PySide2.QtWidgets.PySide2.QtWidgets.QDialogButtonBox.StandardButton) and [Qt6](https://doc.qt.io/qtforpython-6.2/PySide6/QtWidgets/QDialogButtonBox.html#PySide6.QtWidgets.PySide6.QtWidgets.QDialogButtonBox.StandardButton). In Qt5 `StandardButton` is some class, that has `__int__` method, while in Qt6 `StandardButton` is `enum.Flag`, hence the need for `.value`

Reply:
We might want to be compatible with Qt5 here
```py
try:
    # pyside2
    return int(QtWidgets.QDialogButtonBox.Close)
except TypeError:
    # pyside6
    return QtWidgets.QDialogButtonBox.Close.value
```

Reply:
We might want to be Qt5 compatible here

```suggestion
        try:
            # pyside2
            return int(QtWidgets.QDialogButtonBox.Close)
        except TypeError:
            # pyside6
            return QtWidgets.QDialogButtonBox.value
```